src/device/pf_pdport.c

```diff
@@ -294,6 +294,103 @@ int pf_pdport_read_ind (
    return ret;
 }
 
+/**
+ * Run peer check
+ *
+ * Compare llpd peer information with configured pdport peer.
+ * Trigger an alarm if wrong peer is connected.
+ *
+ * @param net              InOut: The p-net stack instance
+ * @param loc_port_num     In:    Local port number.
+ *                                Valid range: 1 .. N, where N is the total
+ *                                number of local ports used by p-net stack.
+ * @param p_lldp_peer_info In:    Peer info
```

Comment:
* @param p_lldp_peer_info In:    Currently received peer info

Reply:
done